client/src/use/useTextLayer.js

```diff
@@ -0,0 +1,52 @@
+import { computed, inject } from '@vue/composition-api';
+
+export default function useTextLayer({
+  filteredDetections,
+  selectedTrackId,
+  typeColorMap,
+}) {
+  // TODO: what's the proper way to consume vuetify in a composition function?
+  const vuetify = inject('vuetify');
+
+  const textData = computed(() => {
+    if (!filteredDetections.value.length) {
+      return null;
+    }
+    const data = [];
```

Comment:
other use{Layers} will do a check for `(filteredDetections.value.length === 0)` I think we may want to make them the same.

- useEventChart.js
- useMarkerLayer.js
We may want to synchronize the return values for these.  Some return an empty array `[]` if the detection length is 0, others will return null instead.